src/lib.rs

```diff
@@ -159,13 +159,13 @@ impl Vob<usize> {
     /// Create a Vob from a `u8` slice. The most significant bit of each byte comes first in the
     /// resulting Vob.
     ///
-    /// If you are running nightly, this method will use the new `reverse_bits` intrinsic.
+    /// If you are running rustc 1.37 or later, this method will use the new `reverse_bits`
```

Comment:
I think we can drop this part of the comment entirely: the performance of the method isn't something we really want to commit to in the comment IMHO.